packages/domain-search/src/components/domain-suggestion-cta/index.tsx

```diff
@@ -41,26 +59,44 @@ export const DomainSuggestionCTA = ( {
 				className="domain-suggestion-cta domain-suggestion-cta--continue"
 				onClick={ handleContinueClick }
 				label={ __( 'Continue' ) }
+				disabled={ disabled || cart.isBusy }
 			>
 				{ compact ? undefined : __( 'Continue' ) }
 			</Button>
 		);
 	}
 
-	const handleAddToCartClick = () => {
-		onClick?.( 'add-to-cart' );
-		cart.onAddItem( uuid );
-	};
+	if ( errorMessage ) {
+		return (
+			// @ts-expect-error open is not a valid prop for the WPDS Tooltip component, but accepted by the underlying Tooltip component.
+			<Tooltip delay={ 0 } text={ errorMessage } placement="top" open>
+				<div className="domain-suggestion-cta-error-container">
+					<Button
+						className="domain-suggestion-cta"
+						isDestructive
+						variant="primary"
+						disabled
+						__next40pxDefaultSize
+						icon={ warning }
+						style={ { flex: 1 } }
```

Comment:
Below, I see a dedicated style for the class "domain-suggestion-cta". We don't need the inline one.

Reply:
We can't apply that rule to that class though as it would change how the other instances of the button would look. I just thought creating a new class name just for that would be too much. But I can do it in a separate PR if we really want this.